LL/sil_birn.trig

```diff
@@ -31,7 +31,8 @@
     a foaf:Person;
     irishRel:genName "Fiachach Fobricc";
     irishRel:nomName "Fiachu Fobrecc";
-    rel:childOf <#AilellaGlais>.
+    rel:childOf <#AilellaGlais>;
+    owl:sameAs <http://example.com/LL/lagin.trig#FiachuFobrecc>.
```

Comment:
FiachuFobrecc is the son of AilellaGlais and not the same as.  I will fix this prior to merge.

Reply:
I was wrong. This is actually correct.  My mistake.